src/sentry/services/hybrid_cloud/notifications/impl.py

```diff
@@ -198,6 +199,24 @@ def serialize_many(
     ) -> List[OpaqueSerializedResponse]:
         return self._FQ.serialize_many(filter, as_user, auth_context)
 
+    def get_subscriptions_for_groups(
```

Comment:
@snigdhas the function name `get_subscriptions_for_groups` seems odd when groups isn't an input, should this be `get_subscriptions_for_projects`?